dags/glam.py

```diff
@@ -294,7 +294,6 @@
         None,
     ),
     task_id="clients_histogram_probe_counts",
-    executor=get_default_executor(),
```

Comment:
Should this be explicitly set to use `SequentialExecutor` e.g. `from airflow.executors.sequential_executor import SequentialExecutor`?

Reply:
`SubDagOperator` defaults to `SeqientialExecutor` so it does not need to be set explicitly. But I'll add a comment so it's clear